cbt.go

```diff
@@ -762,13 +762,14 @@ var commands = []struct {
 		Name: "setgcpolicy",
 		Desc: "Set the garbage-collection policy (age, versions) for a column family",
 		do:   doSetGCPolicy,
-		Usage: "cbt setgcpolicy <table> <family> ((maxage=<d> | maxversions=<n>) [(and|or) (maxage=<d> | maxversions=<n>),...] | never)\n" +
+		Usage: "cbt setgcpolicy <table> <family> ((maxage=<d> | maxversions=<n>) [(and|or) (maxage=<d> | maxversions=<n>),...] | never) [-force]\n" +
```

Comment:
For better or for worse, none of the other parameters in any of the other commands are prefixed with a `-`, so please keep it consistent. Also in terms of placement, I dont think anything else enforces ordering, so I dont this should be required to be the last param. Finally it would be better to document the new param before the gc rules so that it doesnt get lost

Reply:
Removed ordering requirement between GC policy and [force] (but kept table and family, since that's standard across other calls).

Removed prefix + changed order on documentation